backend/README.md

```
@@ -0,0 +1,68 @@
+fastapi dev main.py
+```
+
+Or using uvicorn directly:
+
+```bash
+uvicorn main:app --reload --host 0.0.0.0 --port 8000
+```
+
+The API will be available at:
+
+- **API**: http://localhost:8000
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)